src/main/java/com/pgs/booking/service/UserService.java

```diff
@@ -39,4 +85,16 @@ public UserDetails loadUserDetails(PreAuthenticatedAuthenticationToken token) th
                 .map(Authentication::getPrincipal)
                 .orElseThrow(() -> new UsernameNotFoundException(token.getName()));
     }
+
+    private User getUserById(Long id) {
+        return userRepository.findById(id).orElseThrow(() ->
+                new ResourceNotFoundException("User with id " + id + "not found"));
+    }
+
+    private UserDto getUserDto_UpdateEnabled(Long id, boolean enabled) {
```

Comment:
```suggestion
    private UserDto setUserEnabled(Long userId, boolean enabled) {
```
Normally `_` character is not used in method names. Happens occasionally just for test methods.